### PR TITLE
new plugin "leaflet-geoserver-request"

### PR DESCRIPTION
**Pull request by iamtekson**
This is the plugin for various kind of GeoServer requests on leaflet map. For now, it has only four requests, i.e. wms, wfs, legend, wmsImage.

### CODE REVIEW

**Comment by IvanSanchez** [comment]
But... this looks like (another) WMS/WFS client, not something specific for GeoServer :-/

**Comment by iamtekson** [comment]
Well, I tested the requests with the Geoserver URL. The URL patterns for the request are set for GeoServer. I am not sure it will work for more than GeoServer. For now, it has four types of requests for fetch the GeoServer data, 
* WMS
* WFS
* legend
* wmsImage

I tested the code by putting the `baseLayerUrl` as GeoServer base URL (eg. `http://localhost:8080/geoserver/wms`).

File: docs/plugins.md [hunk 1]
```diff
@@ -4450,6 +4450,16 @@ Ease your development integrating Leaflet into a development framework or automa
 			<a href="https://github.com/moklick">Moritz Klack</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/iamtekson/L.Geoserver">leaflet-geoserver-request</a>
+		</td><td>
+			Geoserver requests in the leaflet map. wms,wfs, legend, wmsImage request on the leaflet. 
+		<a href="https://iamtekson.github.io/L.Geoserver/examples/maps.html">Demo</a>
```
[the remark this answers is not in the document]
Comment (iamtekson) [reply docs/plugins.md:4458]: Hi @Falke-Design, I have updated the demo link of this plugin. Please have a look and please merge it. Lots of people are using this plugin.

[the remark this answers is not in the document]
Comment (Falke-Design) [reply docs/plugins.md:4458]: You need to push your change. The link is still not working

[the remark this answers is not in the document]
Comment (iamtekson) [reply docs/plugins.md]: Sorry, my bad. I just updated the link. Thank you @Falke-Design

**Review by Falke-Design — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**